clang/test/3C/cant_be_nt.c

```diff
@@ -0,0 +1,18 @@
+// Tests for 3C.
+//
+// Checks to make sure _Nt_arrrays only contain pointers & integers
+//
+// : 3c -base-dir=%S -alltypes %s -- | FileCheck -match-full-lines %s
+// RUN: 3c -alltypes -base-dir=%S %s -- | %clang_cc1  -fcheckedc-extension -x c -
+// expected-no-diagnostics
```

Comment:
Please extend with all the standard stuff at the top of tests, and add CHECK, CHECK_ALL etc. below.